package/dart-sass.sh

```diff
@@ -8,12 +8,15 @@
 # package`.
 
 function follow_links() {
-  file="$1"
+  local file="$1"
+
+  cd $(dirname "$file")
   while [ -h "$file" ]; do
     # On Mac OS, readlink -f doesn't work.
     file="$(readlink "$file")"
+    cd $(dirname "$file")
   done
-  echo "$file"
+  echo "$(pwd -P)/$(basename "$file")"
```

Comment:
Can you explain in more detail what this change is doing? This function [is directly copied from the Dart SDK](https://github.com/dart-lang/sdk/blob/master/sdk/bin/dart#L6-L13), so if it's broken in some situation it seems like it should be fixed there too.